src/@utils/nft.ts

```diff
@@ -113,8 +113,6 @@ export async function setNftMetadata(
     []
   )
 
-  console.log('[setNftMetadata]  est Gas set metadata --', estGasSetMetadata)
```

Comment:
having this logged looks like it was intended, but could use the `LoggerInstance` instead

Reply:
well, got used to removing all console.log , since i often forget them 😄